trust-store-trust-policy-specification.md

```diff
@@ -158,23 +184,19 @@ Here is high level uml diagram for signature evaluation:
 
 ## FAQ
 
-**Q1.** Can I scope trust policy by the registry, repository, or namespace?
-
-No, we don't support scoping of trust policy.
-
-**Q2.** How should multiple signatures requirements be represented in the trust policy?
+**Q: How should multiple signatures requirements be represented in the trust policy?**
 
-We don't support n out m signature requirement verification scheme. Validation succeeds if verification succeeds for at least one signature.
+**A:** We don't support n out m signature requirement verification scheme. Validation succeeds if verification succeeds for at least one signature.
 
-**Q3.** Should local revocation and TSA servers are listed in the trust policy to support disconnected environments?
+**Q: Should local revocation and TSA servers are listed in the trust policy to support disconnected environments?**
 
-Not natively supported but a user can configure `revocationValidations` to `skip` and then use extended validations to check for revocation.
+**A:** Not natively supported but a user can configure `revocationValidations` to `skip` and then use extended validations to check for revocation.
 
-**Q4.** Why do we need to include a complete certificate chain(leading to root) in the signature?
+**Q: Why do we need to include a complete certificate chain(leading to root) in the signature?**
```

Comment:
space nit:

**Q: Why do we need to include a complete certificate chain (leading to root) in the signature?**